include/cantera/thermo/Elements.h

```diff
@@ -10,6 +10,8 @@
 
 #include "cantera/base/ct_defs.h"
 
+using namespace std;
```

Comment:
[This SO answer](http://stackoverflow.com/a/5527669/2449192) says to add the qualifier just in the header file. Is that correct then?